lib/services/authentication.dart

```diff
@@ -33,12 +32,15 @@ class Auth implements BaseAuth {
 
   @override
   Future<String> signUp(String email, String password) async {
+    print(email);
```

Comment:
print statements

Reply:
oh yes those can be taken out... good catch